pkg/config/settings.go

```diff
@@ -45,6 +45,9 @@ const (
 var DefaultCr v1.IBMBlockCSI
 
 var DefaultSidecarsByName map[string]v1.CSISidecar
+var DefaultControllerByName map[string]v1.IBMBlockCSIControllerSpec
+var DefaultNodeByName map[string]v1.IBMBlockCSINodeSpec
+var DefaultNamespace string
```

Comment:
it appears these are only used by the test code, so I don't think they belong here